hive/src/main/java/com/netease/arctic/hive/HiveTableProperties.java

```diff
@@ -36,6 +36,9 @@ public class HiveTableProperties {
 
   public static final String ARCTIC_TABLE_PRIMARY_KEYS = "arctic.table.primary-keys";
 
+  // save the location of arctic table. ranger can
+  public static final String ARCTIC_TABLE_ROOT_LOCATION = "arctic.table.root-location";
+
```

Comment:
maybe we cloud use `table.location`   ?   I think we should avoid using  `arctic`

Reply:
in 0.4.x  branch, we still start with arctic.  
In master branch, we can have a deep discussion, also include other properties like arctic.enabled etc.